README.md

```diff
@@ -422,6 +422,7 @@ SCIMetacelloLoadSpec {
   #passwordEnvVar : 'GITHUB_TOKEN',                   // Environment variable containing the password used for authentication
   #useLatestMetacello : true,                         // Upgrade Metacello before loading
   #version : '1.0.0'                                  // Define MC version (for MC Configurations only)
+  #usesIceberg : true				      // Pharo Only | Register the tested repository with Iceberg
```

Comment:
I'm not sure we should call this `#usesIceberg` because it is a bit vague. Should we rename this to `#registerInIceberg`, which seems to be more descriptive if I look at what the `preLoad` hook does. What do you think?

Reply:
I thought something similar, albeit longer than yours which I like.
This is particularly more true with all @theseion corrections.
Okay with you @theseion ?
(I have to modify the text representation of the code because Pharo wants to commit way too much stuff, so I'd rather not redo it a 4th time :D)

Reply:
Yes, I fully agree.